extensions/amp-sidebar/1.0/sidebar.jss.js

```diff
@@ -60,13 +63,21 @@ const defaultBackdropStyles = {
   backgroundColor: 'rgba(0, 0, 0, 0.5)',
 };
 
+//TODO(#32400): See PR description.  This is a workaround dependent on a
```

Comment:
```suggestion
// TODO(#32400): See PR description.  This is a workaround dependent on a
```

Reply:
Did you verify if this is specific to Chrome, or if this workaround is needed in other browsers?

Reply:
Did you verify if this is specific to Chrome, or if this workaround is needed in other browsers?

Reply:
Yes, it appears to be in other browsers as well.

Reply:
Ok, then we should clarify that this should not be removed until verified unneeded in all major browsers that we support, even if the Chromium bug is fixed.

Reply:
Thanks for the review, updated in the PR description!